pkmntypes.py

```diff
@@ -22,3 +22,30 @@ def __init__(self, **kwargs):
 
 class Pokemon():
 	pass
+
+class BattleContext():
+	"""This is a class that describes the point of view of a given Pokemon on the battlefield. It provides
+	enough information for a user to make an informed decision about what turn to make next. 
+
+	:param battle: The state of the battlefield
+	:type battle: dict
```

Comment:
These type don't need to be specified up here...

Reply:
I didn't really know what to do lol I'll delete them 😂  what about param, is that fine?